core/deployment/src/main/java/io/quarkus/deployment/dev/assistant/workspace/AbstractWorkspaceBuildItem.java

```diff
@@ -0,0 +1,90 @@
+package io.quarkus.deployment.dev.assistant.workspace;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import io.quarkus.builder.item.MultiBuildItem;
+
+public abstract class AbstractWorkspaceBuildItem extends MultiBuildItem {
```

Comment:
We definitely want some Javadoc in the build items and the various integration points